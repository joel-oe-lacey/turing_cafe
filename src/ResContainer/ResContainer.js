import React from 'react';
import Res from '../Res/Res';
import './ResContainer.scss';

const ResContainer = ({ reservations }) => {
    const resCards = reservations.map(res => {
        return <Res {...res} />
    })

    return (
        <section className='reservation-container'>
            {resCards}
        </section>
    )
}

export default ResContainer;