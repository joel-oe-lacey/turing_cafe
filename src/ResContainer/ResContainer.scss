.reservation-container {
    height: 70%;
    width: 100%;
    display: flex;
    flex-flow: row wrap;
    justify-content: space-around;
    overflow: scroll;
}