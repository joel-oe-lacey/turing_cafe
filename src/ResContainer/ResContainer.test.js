import React from 'react';
import { shallow } from 'enzyme';
import ResContainer from './ResContainer';

describe('ResContainer', () => {
    const mockReservation = {
        id: '3',
        name: 'Marvin',
        date: '3/14',
        time: '4',
        number: '7'
    }

    it('Should match snapshot', () => {
        const wrapper = shallow(<ResContainer reservations={[mockReservation]} />);
        expect(wrapper).toMatchSnapshot()
    })
})