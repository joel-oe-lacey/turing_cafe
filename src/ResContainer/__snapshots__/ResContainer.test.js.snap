// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResContainer Should match snapshot 1`] = `
<section
  className="reservation-container"
>
  <Res
    date="3/14"
    id="3"
    key="0"
    name="Marvin"
    number="7"
    time="4"
  />
</section>
`;
